docs/CRSF.md

```diff
@@ -40,36 +45,35 @@ Configuration of MAVLink stream rates
 - SRx_POSITION = 2
 - SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
 
-Configuration for CRSF receiver
+Configuration for CRSF Receiver:
 
-Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it can depend on the used board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
+Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
 
 For my Matek H743 board the configuration is:
 
 - BRD_ALT_CONFIG = 1
 - RC_PROTOCOLS = 536 or 512
-- SERIAL7_BAUD = irrelevant
+- RSSI_TYPE = 3
+- SERIAL7_BAUD = Irrelevant (Baud Rate is autodetected by Ardupilot)
 - SERIAL7_OPTIONS = 0
 - SERIAL7_PROTOCOL = 23
 
+There are more options available in the 'RC_OPTIONS' parameter: https://ardupilot.org/plane/docs/parameters.html#rc-options-rc-options
 
 ## mLRS Rx Module Setup
 
-These configurations are not strictly neccesary, but recommended.
+These configurations are not strictly neccesary, but recommended for Ardupilot:
 
 - Rx Out Mode = crsf
+- RX Ser Baudrate = 57600
```

Comment:
maybe add "(not mandatory but strongly recommended)" or similar

Reply:
If the docs earlier state that the ArduPilot MAVLink serial port should be set to 57600 baud, doesn't it make sense to also state the RX Ser Baudrate should be set to 57600?  Don't these need to match?

Reply:
Will add note section as detailed in subsequent comment.